taclib/config.py

```diff
@@ -3,3 +3,6 @@
 
 config = confuse.Configuration("taclib", __name__)
 config["image"] = os.environ.get("TACLIB_IMAGE", config["image"].get(str))
+config["namespaces"]["default"] = os.environ.get(
+    "K8S_NAMESPACE", config["namespaces"]["default"].get(str)
```

Comment:
Please rename the env variable so it is consistent with image -> TACLIB_NAMESPACES__DEFAULT